src/ontology/mondo-edit.obo

```diff
@@ -300967,7 +300969,7 @@ xref: SCTID:266475002 {source="DOID:10211"}
 xref: UMLS:C0008350 {source="MONDO:equivalentTo", source="MONDO:MEDGEN", source="MEDGEN:3039"}
 is_a: MONDO:0004868 {source="PMID:29083691", source="https://orcid.org/0009-0001-6494-4831"} ! biliary tract disorder
 relationship: excluded_subClassOf MONDO:0005281 {source="DOID:10211", source="https://orcid.org/0000-0002-5002-8648"} ! gallbladder disorder
-relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/13887 {source="OMIM:611465"} ! ABCG8
+relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/13887 ! ABCG8
```

Comment:
This entry is related to susceptibility based on presence of the omim phenotype name in `{}` https://omim.org/entry/611465 (see https://omim.org/help/faq#1_6). This is in the [new PR today](https://github.com/monarch-initiative/omim/pull/276/files#diff-261066b07106a3db627bd3a89f5a2a068e3f5af7542e72d04c20bdc978c2b60eR712) and should be kept.

Reply:
I will add this back, using the UI to make a commit is not working this entry.